content/patterns/radio/examples/radio-activedescendant.html

```diff
@@ -234,46 +223,46 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
               <td><code>li</code></td>
               <td>
                 <ul>
-                  <li>Identifies the <code>radio</code> button which is checked.</li>
-                  <li>CSS attribute selectors (e.g. <code>[aria-checked="true"]</code>) are used to synchronize the visual states with the value of the <code>aria-checked</code> attribute.</li>
-                  <li>The CSS <code>::before</code> pseudo-element is used to indicate visual state of checked radio buttons to support high contrast settings in operating systems and browsers.</li>
+                  <li>選択されている <code>radio</code> ボタンを特定。</li>
+                  <li>CSS の属性セレクタ (例: <code>[aria-checked="true"]</code>) を使用して、視覚的な状態と <code>aria-checked</code> 属性の値を同期させる。</li>
+                  <li>オペレーティングシステム及びブラウザのハイコントラスト設定をサポートすべく、CSS の <code>::before</code> 疑似要素が、選択されているラジオボタンの視覚的な状態を示すために使用される。</li>
                 </ul>
               </td>
             </tr>
           </tbody>
         </table>
       </section>
 
+      <section id="at-support">
+        <h2>支援技術のサポート</h2>
+        <iframe
+          class="support-levels-radiogroup-aria-activedescendant"
+          src="https://aria-at.w3.org/embed/reports/radiogroup-aria-activedescendant"
+          height="100"
+          allow="clipboard-write"
+          style="border-style: none; width: 100%;">
+        </iframe>
+      </section>
+
       <section>
-        <h2>JavaScript and CSS Source Code</h2>
+        <h2>JavaScript 及び CSSのソースコード</h2>
```

Comment:
めちゃ細かいですが気づいてしまったので取り込んでいただければ！
```suggestion
        <h2>JavaScript 及び CSS のソースコード</h2>
```